docs/api.md

```diff
@@ -335,6 +335,12 @@ To sum up, the only differences between `puppeteer-core` and `puppeteer` are:
 
 In most cases, you'll be fine using the `puppeteer` package.
 
+But if you still need `puppeteer-core` remember to change the *include* line :-
```

Comment:
let's move this below the "however" paragraph bith bullets: 

When using `puppetee-core`, remember to change the *include* line:
```js
const puppeteer = require('puppeteer-core');
```

Reply:
Doc changed as suggested by you . @aslushnikov